test/dotnet-build.Tests/GivenDotnetBuildBuildsCsproj.cs

```diff
@@ -65,7 +65,7 @@ public void ItRunsWhenRestoringToSpecificPackageDir()
                 .Execute()
                 .Should().Pass();
 
-            var outputDll = Directory.EnumerateFiles(Path.Combine(rootPath, "bin"), "*.dll", SearchOption.AllDirectories).Single();
+            var outputDll = Directory.EnumerateFiles(Path.Combine(rootPath, "bin"), "*.dll", SearchOption.AllDirectories).Single(x => !x.Contains("DiaSymReader"));
```

Comment:
@piotrpMSFT the SDK to pull up to Microsoft.NETCore.App 1.1.1 implicitly isn't in yet